services/skus/controllers.go

```diff
@@ -827,6 +837,8 @@ func getOrderCredsByID(svc *Service, legacyMode bool) handlers.AppHandler {
 				}
 			}
 
+			l.Err(errNotFound).Str("orderID", orderID.String()).Str("itemID", itemIDv.String()).Int("status", status).Msg("error finding creds legacy")
+
 			return handlers.WrapError(err, "Error getting credentials", http.StatusNotFound)
```

Comment:
This is a bug which we will need to fix, the err will be nil as its being taken from the line [here](https://github.com/brave-intl/bat-go/pull/2755/files#diff-e2f505c1ce94c08b8dccd6462d9be724a45ef99e4492c7f708ae21bed918bb02L810)